debug.log

```diff
@@ -6,3 +6,4 @@
 [1125/133808.500:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Do we need this debug log to be checked in? It seems like something you kept locally to debug